lib/franklin_web/endpoint.ex

```diff
@@ -7,7 +7,8 @@ defmodule FranklinWeb.Endpoint do
   @session_options [
     store: :cookie,
     key: "_franklin_key",
-    signing_salt: "XYGfSGw+"
+    signing_salt: "XYGfSGw+",
+    same_site: "Lax"
```

Comment:
TODO: Lookup what this means.

Reply:
Was added in https://github.com/phoenixframework/phoenix/pull/4677 and related to: https://github.com/phoenixframework/phoenix/issues/4877